src/components/SearchBar.jsx

```diff
@@ -7,13 +7,68 @@ const INTENT_VERB = 'OPEN'
 const INTENT_DOCTYPE = 'io.cozy.suggestions'
 const SUGGESTIONS_PER_SOURCE = 10
 
+const normalizeString = str => str.toString().toLowerCase().replace(/\//g, ' ').normalize('NFD').replace(/[\u0300-\u036f]/g, '').split(' ')
+
+const highlightQueryTerms = (searchResult, query) => {
+  const normalizedQueryTerms = normalizeString(query)
+  const normalizedResultTerms = normalizeString(searchResult)
+
+  const matchedIntervals = []
+  const spacerLength = 1
+  let currentIndex = 0
+
+  normalizedResultTerms.forEach(resultTerm => {
+    normalizedQueryTerms.forEach(queryTerm => {
+      const index = resultTerm.indexOf(queryTerm)
```

Comment:
I would prefer a `regex` search

Reply:
That's subjective though. I prefer `indexOf` for searches that don't involve pattern matching.

Reply:
did you benchmark it, I guess regex are way faster than indexOf: https://jsperf.com/regexp-indexof-perf

Reply:
From what I've seen, it depends on the browser and on the test case.
In the test case you've linked, only `test` is used, whereas here we want the index of the match, so `String.search` would be more appropriate. It's also doing the tests on node class names, which we're really not doing here. [This is another benchmark](https://jsperf.com/exec-vs-match-vs-test-vs-search/5) that I just ran, and `indexOf` just blows everything off the water (849,050,703 ops vs 17,738,919 for `search`and `20,586,521` for `test`).

Reply:
ok